advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<p>It may be worth stating that typically you would want to have a variety of different models in an MME, but you're using 100 identical models for the sake of simplicity in this example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3481/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
addressed this in the latest commit